src/util.cpp

```diff
@@ -161,7 +163,7 @@ void Util::parseArgs(int argc,
         return std::string();
 #else
         if (input == "H264") {
-          return "このデバイスは --video-codec=H264 に対応していません。";
+          return "This machine does NOT support --video-codec=H264 option.";
```

Comment:
cannot use --video-codec=H264 option because your device does not have this feature.

Reply:
ditto